packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -291,24 +291,24 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
       case GrowthDirection.forward:
         super.visitChildrenForSemantics((RenderObject child) {
           // The sliver is overlapped at the leading edge.
-          final Offset bottomLeftInViewport = MatrixUtils.transformPoint(
-              child.getTransformTo(parent), child.semanticBounds.bottomLeft
+          final Offset bottomRightInViewport = MatrixUtils.transformPoint(
+              child.getTransformTo(parent), child.semanticBounds.bottomRight
```

Comment:
Here and below: maybe update the comment above to make it more explicit while you're here.
```
// The sliver is overlapped at the leading edge (trailing edge is visible).
```